src/search_algo.hpp

```diff
@@ -196,9 +196,28 @@ prepareScoring(GlobalDataHolder<c_dbIndexType, c_origSbjAlph, c_transAlph, c_red
         seqan::setScoreMatch(context(globalHolder.outfileBlastTab).scoringScheme, options.match);
         seqan::setScoreMismatch(context(globalHolder.outfileBlastTab).scoringScheme, options.misMatch);
 
-        // Seqan3
         globalHolder.scoringScheme.set_simple_scheme(seqan3::match_score{options.match},
                                                      seqan3::mismatch_score{options.misMatch});
+
+        if constexpr (c_redAlph == AlphabetEnum::DNA3BS)
+        {
+            // Seqan2
+            seqan::setScoreBisulfiteMatrix(globalHolder.scoringSchemeAlign, options.match, options.misMatch);
+
+            // Seqan3
+            globalHolder.scoringScheme.set_bisulfite_scheme(seqan3::match_score{options.match},
+                                                            seqan3::mismatch_score{options.misMatch});
+        }
+        else
+        {
+            // Seqan2
+            seqan::setScoreMatch(globalHolder.scoringSchemeAlign, options.match);
+            seqan::setScoreMismatch(globalHolder.scoringSchemeAlign, options.misMatch);
```

Comment:
replace with `globalHolder.scoringSchemeAlign = context(globalHolder.outfileBlastTab)` ?

Reply:
I am unsure this would work .. Doesn't it require the bisulfite scheme to work with all the blast functions that allow only simple scheme?

Reply:
:thinking: 

This if-case is not about BS at all

Reply:
Oh right, I checked the wrong line :see_no_evil:

Reply:
Anyways, this does not work as one is a Blast scoring scheme and the other is not ..